ui/app/components/app/modals/modal.js

```diff
@@ -272,6 +273,35 @@ const MODALS = {
     },
   },
 
+  CUSTOMIZE_METASWAP_GAS: {
+    contents: <SwapsGasCustomizationModal />,
+    mobileModalStyle: {
+      width: '100vw',
+      height: '100vh',
+      top: '0',
+      transform: 'none',
+      left: '0',
+      right: '0',
+      margin: '0 auto',
+    },
+    laptopModalStyle: {
+      width: 'auto',
+      height: '0px',
+      top: '80px',
+      left: '0px',
+      transform: 'none',
+      margin: '0 auto',
+      position: 'relative',
+    },
+    contentStyle: {
+      borderRadius: '8px',
+    },
+    customOnHideOpts: {
+      action: resetCustomGasData,
```

Comment:
This seems wrong :thinking: We don't want to reset custom gas data when this modal is closed anymore.

Do we want to reset the swaps gas data when this happens? Or do nothing? I would guess that doing nothing is fine.

Reply:
Removed in 8b406b2a4